homework-3/src/views/PatientProfile.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import BoxRow from '../components/common/Boxes/StylBoxRow';
+import BoxFilter from '../components/Boxes/BoxFilter';
+import BoxRowPadding from '../components/common/Boxes/StylBoxRowPadding';
+import BoxPatients from '../components/common/Boxes/StylBoxPatients';
+import BoxDoctor from '../components/common/Boxes/StylBoxDoctor';
+import BoxFlexColumn from '../components/common/Boxes/StylBoxFlexColumn';
+import BoxPatientContent from '../components/common/Boxes/StylBoxPatientContent';
+import BoxBtnPatients from '../components/common/Boxes/StylBoxBtnPatients';
+import BoxPatientsList from '../components/common/Boxes/StylBoxPatientsList';
+import TitleName from '../components/common/Titles/StylTitleName';
+import TitlePatients from '../components/common/Titles/StylTitlePatients';
+import SubTitlePostion from '../components/common/Titles/StylSubTitlePosition';
+import AvatarActive from '../components/AvatarActive';
+import BtnPatients from './../components/BtnsComponents/BtnPatients';
+import BtnCreateAppointment from './../components/BtnsComponents/BtnCreateAppointment';
+import BtnPatientsHiddenMob from './../components/BtnsComponents/BtnPatientsHiddenMob';
+import CardPatient from '../components/CardPatientProfilePatient/CardProfilePatient';
+import Loader from './../components/Loader/Loader';
+
+import patient from '../img/avatars/avatar-patient.png';
+import avatar2 from '../img/avatars/patients-13.png';
+import logo from '../img/logo.png';
+
+const DoctorProfile = ({ title }) => {
+  React.useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
+
+      setLoading((loading) => !loading);
+    };
+
+    loadData();
+  }, []);
+
+  if (loading) {
+    return <Loader />;
+  } else {
+    return (
+      <BoxPatients>
+        <BoxRow>
+          <img src={logo} alt='logo' />
+          <BoxDoctor>
+            <BoxFlexColumn>
+              <TitleName>Larry Prinston</TitleName>
+              <SubTitlePostion>Patient</SubTitlePostion>
+            </BoxFlexColumn>
+            <AvatarActive src={patient} alt='avatar-user' />
+          </BoxDoctor>
+        </BoxRow>
+        <BoxPatientContent>
+          <BoxBtnPatients>
+            <BtnPatients active={false} text='Profile' />
+            <BtnPatients active={true} text='Appointments  ' />
+            <BtnPatientsHiddenMob active={false} text='Resolutions  ' />
+          </BoxBtnPatients>
+          <BoxRowPadding>
+            <TitlePatients>My Appointments</TitlePatients>
+            <BoxDoctor>
+              <BoxFilter text='Show: ' filter='Upcoming' />
+              <BtnCreateAppointment />
+            </BoxDoctor>
+          </BoxRowPadding>
+          <BoxPatientsList>
+            <CardPatient
```

Comment:
code become clearer if u put all your proprs information about patient in db.json and u can leave only one props data={data}
in components CardPatient u can map this `data `props and create templates for your future cards
for Example :  `const CardPatient = ({data}) => {
   return (
      <ul>
           {data.map(({name,statusText,data,alt,img,info}) => {
               and here u create one template where u put all info from data
           })
      </ul>
   )
}`

Reply:
Created 2 files `dbProfileDoctor` and `dbProfilePatient`